airflow/dag_processing/manager.py

```diff
@@ -1013,6 +994,15 @@ def collect_results(self) -> None:
 
         self.log.debug("%s file paths queued for processing", len(self._file_path_queue))
 
+    @staticmethod
+    def _create_process(file_path, pickle_dags, dag_ids, callback_requests):
+        """Creates DagFileProcessorProcess instance."""
+        from airflow.dag_processing.processor import DagFileProcessorProcess
```

Comment:
I assume this was put here to avoid a circular import, but did you profile at all to see if there is a performance impact?

Reply:
I will profile it

Reply:
With https://github.com/apache/airflow/pull/16816 now merged, the cyclic import is gone and this can just be a normal top level import.